content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -71,7 +71,7 @@ A toleration "matches" a taint if the keys are the same and the effects are the
 * the `operator` is `Exists` (in which case no `value` should be specified), or
 * the `operator` is `Equal` and the `value`s are equal
 
-`Operator` defaults to `Equal` if not specified.
+`operator` defaults to `Equal` if not specified.
```

Comment:
@sftim , This section is confusing.
This is a new sentence (capitalized previously)?
Instead of starting with `operator`, could modify to:
The `operator` defaults to ... OR
If not specified, `operator` defaults to `Equal`.
What do you think?

Reply:
Reworded